Mathlib/Data/Matrix/Basis.lean

```diff
@@ -216,6 +216,44 @@ theorem mul_of_ne {k l : n} (h : j ≠ k) (d : α) :
 
 end
 
+section Commute
+
+variable [Fintype n]
+
+theorem comm_stdBasisMatrix {i j k : n} {M : Matrix n n α}
+    (hM : stdBasisMatrix i j 1 * M = M * stdBasisMatrix i j 1) (hkj : k ≠ j) : M j k = 0 := by
```

Comment:
A little more concise as
```suggestion
    (hM : Commute (stdBasisMatrix i j 1) M) (hkj : k ≠ j) : M j k = 0 := by
```
And below

Reply:
Thanks! I am really sorry that the APP didn't notify me.

Reply:
Sorry, I now realize that this comment was incorrect; your original spelling was actually better, because it generalizes to rectangular matrices!